### DISCUSSION TITLE
Why Can't you open a web page after executing “tgcf-web”?

### DISCUSSION THREAD

**Discussion by CaiGeen** [aahnik/tgcf#383] [Q&A] [answered]
Is this about the agency? 

Or is there something I haven't noticed? Thank you for your trouble. If anyone can help me, thank you very much.

<img width="681" alt="微信截图_20221205163746" src="https://user-images.githubusercontent.com/9194254/205591475-57acecc8-4161-4a32-92c9-c62d2d1255f0.png">
<img width="970" alt="微信截图_20221205163756" src="https://user-images.githubusercontent.com/9194254/205591488-6f42bd51-7602-48ee-8de7-764417f1db2b.png">

**Comment by aahnik** [comment, marked as answer]
this is due to some mistakes in your network configuration. its not possible to say anything, without knowing your system, and setup. feel free to chat with the community https://telegram.me/aahniks_code_discussion
